lib/perl/Genome/Model/Tools/Vcf/EvaluateVcfs.pm

```diff
@@ -0,0 +1,369 @@
+        },
+
+        pass_filter_expression => {
+            is => "Text",
+            doc => "String to pass to vcflib vcffilter in order to "
+                   . "select variants from the evaluation VCF",
+            default_value => q{-g 'FT = PASS | FT = .'},
+        },
+    ],
+
+    has_output => [
+        rawdata => {
+            is => "HASH",
+            doc => "The organized stats & metadata generated during "
+                   . "primary execution",
+        },
+    ],
+};
+
+sub execute {
+    my $self = shift;
+    my $configs = $self->parse_config_file();
+    $self->collect_statistics($configs);
+    $self->rawdata($configs);
+    $self->display_summary_stats();
+    return 1;
+}
+
+sub display_summary_stats {
+    my $self = shift;
+    my $configs = $self->rawdata;
+
+    $self->status_message("Generating Summary Report");
+
+    print "\n";
+    print "==================", "\n";
+    print "Valdiation Summary", "\n";
+    print "==================", "\n";
+    print "\n";
+
+    my @stat_names = $self->stat_types;
+    my @headers = ('Name', 'ID', 'VarType', @stat_names);
+    print join("\t", @headers), "\n";
+
+    for my $set (@{$configs}) {
+        my ($name, $id, $type, $stats) =
+          @{$set}{'name', 'id', 'variant_type', 'stats'};
+        my @results = @{$stats}{@stat_names};
+        print join("\t", $name, $id, $type, @results), "\n";
+    }
+}
+
+sub collect_statistics {
+    my ($self, $inputs) = @_;
+
+    my ($i, $total) = (1, scalar @{$inputs});
+    $self->status_message(
+        "Performing Evaluations & Collecting Statistics (%d vcfs)",
+        $total
+    );
+
+    for my $set (@{$inputs}) {
+        $self->status_message(
+            "[ $i | $total ] Name: %s / Model ID: %s / Type: %s",
+            $set->{'name'}, $set->{'id'}, $set->{'variant_type'}
+        );
+        $self->status_message(" ---> Vcf: %s", $set->{'vcf'});
+
+        my $wkspace = $self->create_subdirectory_workspace($set);
+        $self->status_message(" ---> Output Workspace: %s", $wkspace->stringify);
+
+        $self->status_message(" ---> Start VCF Evaluation");
+        my $stats = $self->_run_evaluate_vcf($wkspace, $set);
+        $self->status_message(" ---> Finish VCF Evaluation\n\n");
+
+        $set->{'stats'} = $stats;
+        $i++;
+    }
+}
+
+sub parse_config_file {
+    my $self = shift;
+    my $config = $self->fetch_config_file;
+
+    $self->status_message("Parsing configuration file : $config");
+
+    my $fh = $config->openr;
+
+    my @inputs;
+    while (my $line = <$fh>) {
+        chomp $line;
+        my ($name, $id, $path, $variant_type, $sample) = split("\t", $line);
+        $self->validate_type($variant_type);
+        my $gold_vcf =
+          $variant_type eq "snvs" ? $self->gold_snv_vcf : $self->gold_indel_vcf;
+
+        my $clean_indels = $variant_type eq 'indels' ? 1 : 0;
+
+        my $model = Genome::Model->get($id);
+        my $build = $model->last_succeeded_build;
```

Comment:
Would it make sense for this tool to support being passed builds instead of models?  (It could try both by doing something like:
```perl
my $build = Genome::Model::Build->get($id) || Genome::Model->get($id)->last_succeeded_build;
```

Regardless it might be good to make sure we got a `$model` or a `$build` rather than dying with a `Can't call method ... on an undefined value` error.